src/test/java/executor/service/service/ExecutionServiceFacadeTest.java

```diff
@@ -0,0 +1,43 @@
+package executor.service.service;
+
+import executor.service.model.ProxyConfigHolderDTO;
+import executor.service.model.ProxyCredentialsDTO;
+import executor.service.model.ProxyNetworkConfigDTO;
+import executor.service.model.ScenarioDTO;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.openqa.selenium.WebDriver;
+
+@ExtendWith(MockitoExtension.class)
+class ExecutionServiceFacadeTest {
+    @Mock
+    ScenarioExecutorService scenarioExecutorService;
```

Comment:
```suggestion
    private ScenarioExecutorService scenarioExecutorService;
```
let's use common code style